client/state/ui/selectors.js

```diff
@@ -90,3 +90,33 @@ export function isPreviewShowing( state ) {
 export function getPreviewUrl( state ) {
 	return state.ui.currentPreviewUrl;
 }
+
+/**
+ * Returns the current layout focus area
+ *
+ * @param  {Object}  state Global state tree
+ * @return {?String}  The current layout focus area
+ */
+export function getCurrentLayoutFocus( state ) {
+	return get( state, 'ui.layoutFocus.current' );
```

Comment:
Hm... it was last week so I'm having trouble remembering, but there was some situation where it wasn't available. I think maybe it's when these selectors are used by the boot code before the Redux store is initialized. This way they safely return null.